files/docker/Dockerfile

```
@@ -3,6 +3,11 @@
 FROM fedora:31
 
 ARG SOURCE_BRANCH
+RUN  if [[ -z $SOURCE_BRANCH ]]; then \
+echo -e "\nSOURCE_BRANCH build argument have to be defined using \
+\"--build-arg key=val\" option.\nIt is branch which should be used for installing internal \
+projects (e.g. ogr, packit).\n" && exit 1;\
+fi
```

```suggestion
echo -e "\nMissing SOURCE_BRANCH build argument! Please add \
\"--build-arg SOURCE_BRANCH=<val>\" to the build command to specify it!\n\
This is the branch used when installing other Packit projects (e.g. ogr, packit).\n" && exit 1;\
fi
```

(use simple sentences, avoid complicated  grammatical structures, _present simple_ is your friend :slightly_smiling_face: )

---

fixed, thanks for suggestion